src/component.ts

```diff
@@ -0,0 +1,41 @@
+import type { IHttpServerComponent } from '@well-known-components/interfaces'
+import type { ITracerComponent, TraceContext } from '@well-known-components/tracer-component'
+import { parseTraceParentHeader, parseTraceState } from './logic'
+import { IHttpTracerComponent } from './types'
+
+export function createHttpTracerComponent(components: {
+  server: IHttpServerComponent<IHttpServerComponent.DefaultContext<any>>
```

Comment:
Is it possible to type this instead of using any?

Reply:
Done!